pypeman/tests/test_nodes.py

```diff
@@ -385,6 +386,8 @@ def test_ftp_nodes(self):
             fake_ftp.upload_file.assert_called_once_with('test_write.part', 'message_content')
             fake_ftp.rename.assert_called_once_with('test_write.part', 'test_write')
 
+    @unittest.skipIf((sys.version_info.major == 3 and sys.version_info.minor == 7),
+                     "difficulty to mock async with statement in py3.7")  # TODO: rm in py3.8+
```

Comment:
`sys.version_info[:2] == (3, 7)` does the same and is shorter

Reply:
done